lib/v2/screens/explore_screens/vote_screens/proposal_details/components/proposal_details_middle.dart

```diff
@@ -0,0 +1,149 @@
+
+class ProposalDetailsMiddle extends StatelessWidget {
+  const ProposalDetailsMiddle({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<ProposalDetailsBloc, ProposalDetailsState>(
+      buildWhen: (previous, current) => previous.currentIndex != current.currentIndex,
+      builder: (context, state) {
+        return Column(
+          children: [
+            Padding(
+              padding: const EdgeInsets.all(16.0),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  Row(
+                    children: [
+                      Flexible(
+                        child: Text(
+                          state.proposals[state.currentIndex].title,
+                          style: Theme.of(context).textTheme.headline7,
+                        ),
+                      )
+                    ],
+                  ),
+                  const SizedBox(height: 10.0),
+                  Text(
+                    state.proposals[state.currentIndex].summary,
+                    style: Theme.of(context).textTheme.subtitle3OpacityEmphasis,
+                  ),
+                  const SizedBox(height: 30.0),
+                  Text('Created by'.i18n, style: Theme.of(context).textTheme.subtitle2),
+                  const SizedBox(height: 10.0),
+                  Row(
+                    children: [
+                      const ProfileAvatar(size: 60, account: 'TODO'),
+                      Expanded(
+                        child: Padding(
+                          padding: const EdgeInsets.only(left: 10.0),
+                          child: Column(
+                            children: [
+                              Row(
+                                children: [
+                                  Expanded(
+                                    child: Text(
+                                      state.proposals[state.currentIndex].creator,
+                                      style: Theme.of(context).textTheme.button,
+                                    ),
+                                  ),
+                                  Text(
+                                    state.proposals[state.currentIndex].createdAt,
+                                    style: Theme.of(context).textTheme.subtitle3,
+                                  ),
+                                ],
+                              ),
+                              const SizedBox(height: 10.0),
+                              Row(children: [Text('TODO', style: Theme.of(context).textTheme.subtitle2)]),
+                            ],
+                          ),
+                        ),
+                      )
+                    ],
+                  ),
+                ],
+              ),
+            ),
+            const DividerJungle(),
+            Padding(
+              padding: const EdgeInsets.all(16.0),
```

Comment:
Hi Raul, Please use the Horizontal padding const when applicable

Reply:
done